src/code42cli/cmds/enums.py

```diff
@@ -0,0 +1,8 @@
+class OutputFormat:
+    ASCII = "ASCII-TABLE"
```

Comment:
for the user's sake, lets call this `table` instead of `ascii-table`.

Reply:
Many tables are ascii tables.

Reply:
Done.